### PR TITLE
Update remote-help.md - Fix formatting on note

### PR DESCRIPTION
**Pull request by colealbrecht**
Fix formatting on !Note under install and update

### CODE REVIEW

**Comment by PRMerger13** [comment]
@colealbrecht : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/remote-actions/remote-help.md [hunk 1]
```diff
@@ -127,9 +127,9 @@ For users that opted out of automatic updates, when an update to remote help is
 - Intune admins can download and deploy the app to enrolled devices. For more information about app deployments, see [Install apps on Windows devices](../apps/apps-windows-10-app-deploy.md#install-apps-on-windows-10-devices).
 - Individual users who have permissions to install apps on their devices can also download and install remote help.
 
-[!NOTE]
-- On April 5th, 2022, existing users of remote help will see a recommended upgrade screen when they open the remote help app. Users will be able to continue using remote help without upgrading. 
-- On April 12th, 2022, existing users of remote help will see a mandatory upgrade screen when they open the remote help app. They will not be able to proceed until they upgrade to the latest version of remote help.
+> [!NOTE]
+> - On April 5th, 2022, existing users of remote help will see a recommended upgrade screen when they open the remote help app. Users will be able to continue using remote help without upgrading. 
+> - On April 12th, 2022, existing users of remote help will see a mandatory upgrade screen when they open the remote help app. They will not be able to proceed until they upgrade to the latest version of remote help.
```
Comment (JohanFreelancer9) [inline memdocs/intune/remote-actions/remote-help.md:132]:
```suggestion
> - On April 5, 2022, existing users of remote help will see a recommended upgrade screen when they open the remote help app. Users will be able to continue using remote help without upgrading. 
> - On April 12, 2022, existing users of remote help will see a mandatory upgrade screen when they open the remote help app. They will not be able to proceed until they upgrade to the latest version of remote help.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @colealbrecht, kindly commit the additional suggested changes. Many thanks.

**Comment by PRMerger8** [comment]
@colealbrecht : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by Smritib17 — APPROVED** [review]

**Pull request merged**